ui/src/app/files/components/file-overview/file-overview.component.ts

```diff
@@ -22,6 +22,7 @@ import { MatTableDataSource } from '@angular/material/table';
 import { MatPaginator } from '@angular/material/paginator';
 import { ConfirmDialogComponent } from '@streampipes/shared-ui';
 import { MatDialog } from '@angular/material/dialog';
+import * as FileSaver from 'file-saver';
```

Comment:
Just one minor comment. Maybe we can replace this with `import { saveAs } from 'file-saver';` 
I saw that we use the file-saver like you did it in `PipelinesComponent` as well, but I guess it is better to import it without the `*`.

Reply:
Thanks! Updated the import statement